samples/XCT.Sample/Pages/Behaviors/MaskedBehaviorPage.xaml

```diff
@@ -71,6 +71,19 @@
                 </Entry.Behaviors>
             </Entry>
             <Label Padding="10,0,10,50" FontSize="Small" Text="e.g. Phone Number '(XXX) XXX-XXXX'" />
+
+            <Entry x:Name="PhoneEntry"
+                   Keyboard="Telephone"
+                   Placeholder="Phone">
+                <Entry.Behaviors>
+                    <xct:CharactersValidationBehavior x:Name="PhoneValidator"
+                                                      CharacterType="Digit"
+                                                      Flags="ValidateOnValueChanging"
+                                                      MinimumCharacterCount="11" />
+                    <xct:MaskedBehavior Mask="+1 (AAA) AAA-AAAA" UnMaskedCharacter="A" />
+                </Entry.Behaviors>
+            </Entry>
+            <Label Padding="10,0,10,50" FontSize="Small" Text="Test for issue #1276'" /> <!-- https://github.com/xamarin/XamarinCommunityToolkit/issues/1276 -->
```

Comment:
We have "Test Cases" folder for this purpose :)

Reply:
@AndreiMisiukevich lol never noticed it, will look at moving it